_docs/for-contributors/2017-01-02-implementation.md

```diff
@@ -190,6 +190,29 @@ Broadcast messages are resent to neighbors right after retrieval (before
 handling). Also, they are being checked against LRU cache, and messages
 that have been already received once get ignored.
 
+### Leaders and rich men computation (LRC)
+"Slot leaders" and "rich men" are two important notions of Ouroboros Proof of Stake Algorithm:
+
+- Slot leaders: A slot leader is a such node which creates block in the current slot.
```

Comment:
I think `Slot leaders` should be, but `rich men` probably not since it's a bit too protocol-specific.